heroku_connect/management/commands/import_mappings.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from django.core.management import BaseCommand, CommandError
+
+from heroku_connect import utils
+from heroku_connect.conf import settings
+
+
+class Command(BaseCommand):
+    """Import Heroku Connect mappings to connection."""
+
+    help = __doc__.strip()
+
+    def add_arguments(self, parser):
+        parser.add_argument('--app', '-a', dest='HEROKU_APP', type=str,
```

Comment:
app_name should be mandatory. Lets add `required=True`  and also put the `default=settings.HEROKU_CONNECT_APP_NAME`

Reply:
https://docs.python.org/3/library/argparse.html#required

Reply:
If you do that it will actually try to access the setting. It does make more sense to add this to the help text.

Reply:
Anyhow, it's also not required. To perform the upload only the connection id is required.

Reply:
If you have multiple connections per app, you need to provide an ID. The app would be not a required information.